dev/appveyor-install-dependencies.ps1

```diff
@@ -95,7 +95,7 @@ $env:MAVEN_OPTS = "-Xmx2g -XX:ReservedCodeCacheSize=512m"
 Pop-Location
 
 # ========================== Hadoop bin package
-$hadoopVer = "2.6.4"
+$hadoopVer = "2.7.3"
```

Comment:
Sean, I mean 2.7.1 :) looks 2.7.3 wjnutils is not there.

Reply:
Oh I see. I added a note here instead.